Components/alert.css

```diff
@@ -0,0 +1,60 @@
+.alert {
+  padding: 1rem 0.5rem;
+  margin-bottom: 1rem;
+}
+.alert:last-of-type {
+  margin-bottom: 0;
+}
+.alert-primary {
+  background-color: #60a5fa;
+  border-radius: 0.2em;
+  color: #1e3a8a;
+}
+.alert-secondary {
+  background-color:  var(--dark-secondary-color);
+  border-radius: 0.2em;
+  color: var(--dark-secondary-text-color);
+}
+.alert-success {
+  background-color: #4ade80;
+  border-radius: 0.2em;
+  color: #14532d;
+}
+.alert-warning {
+  background-color: #fde68a;
+  border-radius: 0.2em;
+  color: #e09004;
```

Comment:
use single style for color eg: if you are using hex code than use that at all places

Reply:
you can use variables for giving colors put that variable inside :root

Reply:
Added variables after review. Thanks for the review!